microraiden/microraiden/test/test_proxy.py

```diff
@@ -8,6 +8,9 @@
 from microraiden.proxy.resources import Expensive
 from microraiden.proxy.paywalled_proxy import PaywalledProxy
 
+import logging
```

Comment:
What's this for?

Reply:
I had some logging enabled there, I'd leave it there, we'll soon need logs to investigate slow proxy tests.